checks.d/http_check.py

```diff
@@ -143,6 +143,12 @@ class HTTPCheck(NetworkCheck):
 
     def __init__(self, name, init_config, agentConfig, instances):
         self.ca_certs = init_config.get('ca_certs', get_ca_certs_path())
+        self.proxy_host = agentConfig.get('proxy_host','')
+        self.proxy_port = agentConfig.get('proxy_port','')
+        self.proxies = {}
```

Comment:
i think self.proxies should be None if proxy settings are not set, so that requests uses its default settings.

Also looks like you are not taking into account the username and password.

Also you shouldn't read agentConfig yourself but likely call  `util.proxy.get_proxy` https://github.com/DataDog/dd-agent/blob/master/utils/proxy.py#L26